### ISSUE TITLE
bug: sameword-numbering broken in endnotes with memorybreaks

### ISSUE THREAD

**Issue by floriandk** [maieul/ledmac#831]
# Version
occurs on up-to-date reledmac 2019/11/28 v2.32.2, but the problem has obviously been there for years unnoticed (tested on TL17 reledmac 2018/02/28 v2.26.6)

# Regression

# Description

Working with very long consecutive texts without chapterbreaks the suggestion of the `reledmac`-manual §5.2.7 to introduce memorybreaks is still valid, as it drastically reduces compilation time (8.5min vs 45min for one of my projects; XeLaTeX will even stall because of memory-limitations). 

But the `sameword`-mechanism is broken when using endnotes: Identical `edtext`s will only be annotated correctly in the p-sections directly before and after the last `\memorybreak`.

There is logged *one* error irregardless of how many p-sections there are:

```
./memorybreakbug.Aend:27: Missing number, treated as zero.
<to be read again> 
}
l.27 [1]{cc}}{A}{A}{}{L}{edtxt@3}
```

# Minimal Working Example

```latex
\documentclass{scrartcl}

\usepackage[series={A},nofamiliar,noeledsec,noledgroup,draft]{reledmac}

\newcommand{\memorybreak}{\pausenumbering\resumenumbering}
%%the sameword-numbering is correct when using footnotes instead:
%\let\Aendnote=\Afootnote


\begin{document}

\beginnumbering
\pstart
aa bb %
\edtext{\sameword[1]{cc}}{%
	\Aendnote{A}} dd aa bb \sameword{cc} dd
\pend\memorybreak
\pstart
aa bb %
\edtext{\sameword[1]{cc}}{%
	\Aendnote{A}} dd aa bb \sameword{cc} dd
\pend\memorybreak
\pstart
aa bb %
\edtext{\sameword[1]{cc}}{%
	\Aendnote{A}} dd aa bb \sameword{cc} dd
\pend\memorybreak
\pstart
aa bb %
\edtext{\sameword[1]{cc}}{%
	\Aendnote{A}} dd aa bb \sameword{cc} dd
\pend
\endnumbering

\doendnotes{A}

\end{document}
```

**Comment by maieul** [comment]
yes, of course. It is logical. Unfortunalty, I am to busy, yet, to look
for this bug (and more generally, for reledmac)

**Comment by floriandk** [comment]
You wouldn't have time to look into this, would you? The compilation time of my project without memorybreaks is killing me...

**Comment by maieul** [comment]
sorry, no time. My great mother is dead, and I have a big project on the 
fire.

Le 14.03.22 à 12:56, Florian Grammel a écrit :

**Comment by floriandk** [comment]
Toutes mes condoléances! I'll get by for now and hope for when you'll again have time time to spare :)

**Comment by floriandk** [comment]
Salut!
Sorry to be a bother but as I am working on this huge file again I though I might try to bump this. To get reledpar to find its balance on 600 pages can literally take days without memorybreaks.
Perhaps you have some time now?
